content/c/concepts/basic-output/basic-output

```diff
@@ -0,0 +1,71 @@
+---
+Title: "Basic Output"
+Subjects:
+  - "Code Foundations"
+  - "Computer Science"
+Tags:
+  - "Print"
+  - "Output"
+---
+
+In the C language basic console output is achieved using functions from the standard input output header file (stdio.h)
```

Comment:
Bold **stdio.h** with `**` to follow Codecademy Curriculum standards.

Reply:
Done